src/freertos_drivers/net_freertos_tcp/FreeRTOSTCPSocket.cxx

```diff
@@ -274,7 +277,7 @@ int FreeRTOSTCPSocket::accept(
 
     Socket_t sd = FreeRTOS_accept(s->sd, &fr_address, &fr_address_len);
```

Comment:
The present implementation in FreeRTOS_accept will store the port and source address in the passed freertos_sockaddr structure if non-null with no further checks.  The API documentation for FreeRTOS_accept indicates that at the present the length parameter is ignored, however should be set to the size of the freertos_sockaddr structure; no mention is made of conditioning the structure itself.